nextest-runner/default-config.toml

```diff
@@ -52,8 +52,10 @@ success-output = "never"
 # to false.
 fail-fast = true
 
-# Treat a test that takes longer than this as slow, and print a message.
-slow-timeout = "60s"
+# Treat a test that takes longer than this period as slow, and print a message.
+# Given a non-zero positive integer the test will be shutdown the number periods
```

Comment:
I like `period` but let's rename it `kill-after` to make it clear to users what happens.

Reply:
That sounds good to me.